api/v1/serializers/instance_serializer.py

```diff
@@ -51,6 +52,14 @@ def __init__(self, *args, **kwargs):
         self.request_user = user
         super(InstanceSerializer, self).__init__(*args, **kwargs)
 
+    def get_size(self, obj):
```

Comment:
Yes, so `get_size` gets called when we have`size = SerializerMethodField` (Line 26) it will call `def get_<field-name>`